Source/options.js

```diff
@@ -453,5 +453,39 @@ $(function() {
     save('ngramMax', 'Maximum word sequence length saved');
   }
 
+  // Listener to update blacklist from context menu
+  chrome.runtime.onMessage.addListener(function(request, sender, sendResponse) {
```

Comment:
I think I understand the reason why you added this listener to "options.js", but "background.js" might be a better place. "Options.js" will only be executed when the user opens the options page. Therefore, it seems that until the user opens the options page, the messages sent in line 220 of "background.js" will not be handled, because no listener will have been registered yet. Isn't this is a bug? I haven't tested it.